content/projects/basic-unit-test-mocks-kata/_index.md

```diff
@@ -153,4 +153,17 @@ then they are wrong.
 - You can make the emails that get sent look good as well. it doesn't just need to be plain text
 - You can look into sending sms instead of email
 - link to the quote author's wikipedia page straight from the email/sms
-- add extra information such as a weather report
\ No newline at end of file
+- add extra information such as a weather report
+
+
+**Instructions for reviewer**
+
+- Please pay attention to how the unit tests are written:
+
+> The learner should demonstrate that the code does what it should and does it efficiently.
+
+> They should test correctly. Please refer to [Jasmine spies](https://umuzi-org.github.io/tech-department/topics/jasmine-spies/).
```

Comment:
please never ever hardcode urls in a web project when they can be calculated.  Also, the url is incorrect. This is why we dont hardcode stuff like that. If that url works then it would be surprising. And possibly outdated because that isn't where we host this site.

Please make use of the contentlink shortcode. If you search for it in the repo you will see how it is used. (Use Ctrl+Shift+F to search all the files if you are using vscode.